### DISCUSSION TITLE
Empty project list

### DISCUSSION THREAD

**Discussion by hiaw** [LazyVim/LazyVim#1978] [Q&A] [answered]
So I've added the following to my `lazy.lua` config file and I can see the project option in the dashboard and in the file menu. So it means it's loading that plugin correctly. But there is no projects in the list when I open it up. I also don't see "project" as one of the option when I open run `:Telescope`. I had previous installation on an old laptop that'll add any older with `.git` folder as a project when I open it. 
```    
{ import = "lazyvim.plugins.extras.util.project" },
```

I just discovered that I can do it manually by using `:AddProject` from the root folder of my project. But ideally it happens automatically.

**Comment by dpetka2001** [comment, marked as answer]
Create a personal spec for `project.nvim` and put `manual_mode = false`. It should autodetect then.
